apps/challenges/serializers.py

```diff
@@ -48,13 +65,6 @@ class Meta:
                   'is_public', 'is_active', 'codename')
 
 
-class DatasetSplitSerializer(serializers.ModelSerializer):
```

Comment:
I agree that it can confuse other people in future. I will reorder it as  previously and yes definitely we can open an issue in future to reorder them.